### PR TITLE
feat: o-meter demos for storybook

### PR DESCRIPTION
**Pull request by akomiqaia**

### CODE REVIEW

File: components/o-meter/src/tsx/meter.tsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+}
+
+export type MeterProps = {
+	meterValue: number;
+	ariaLabel: string;
+	min?: number;
+	max?: number;
+	low?: number;
+	high?: number;
+	optimum?: number;
```
[the remark this answers is not in the document]
Comment (akomiqaia) [reply components/o-meter/src/tsx/meter.tsx:22]: as discussed yesterday this props work together and there is not set that we need to give to o-metere component

File: components/o-meter/stories/meter.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+};
+
+export const customisedColors: ComponentStory<typeof Meter> = Story.bind({});
+customisedColors.storyName = 'Meter With Customised Colours';
+
+customisedColors.args = {
+	ariaLabel: 'demo meter 80',
+	meterValue: 80,
+	customColors: {
+		'--o-meter-background-color': 'lightgray',
```
[the remark this answers is not in the document]
Comment (akomiqaia) [reply components/o-meter/stories/meter.stories.tsx:76]: does this apply on Custom dimensions as well?

File: components/o-meter/src/tsx/meter.tsx [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+	valueBox?: boolean;
+};
+
+const MeterContainerWrapper = ({condition, details, children}) => {
+	const wrapper = children => (
+		<div className="o-meter-container" style={details.customDimensions}>
```
[the remark this answers is not in the document]
Comment (akomiqaia) [reply components/o-meter/src/tsx/meter.tsx:14]: This is a conditional wrapper and not sure how we can remove it. It seems like we apply this wrapper when we have value box. I guess we can use ternary or another if/else in `Meter`. I can make it in a way that we pass less props down or do something like this maybe? 

```jsx
valueBox ? 
(<div className="o-meter-container">
  <meter> ... </meter>
  <span className="o-meter-value"
	aria-hidden="true"
	style={{left: boxValueIndentation}}>
	{value}
  </span>
</div>) : 
<meter>....</meter>
```

or does `o-meter-container` existence does not really matter for `<meter>`?

[the remark this answers is not in the document]
Comment (notlee) [reply components/o-meter/src/tsx/meter.tsx]: Resolved, we discussed this one in person: https://github.com/Financial-Times/origami/pull/751/commits/79a576b1cc10a01bfc674836ed2019d6281ed46e

**Review by notlee — APPROVED** [review]

**Pull request merged**